generators/client-2/templates/src/main/webapp/app/account/_account.module.ts

```diff
@@ -49,16 +39,6 @@ angular
         'ui.bootstrap',
         'ui.router'
     ])
-    .config(AccountStateConfig)
-    .config(ActivateStateConfig)
-    .config(RegisterStateConfig)
-    .config(PasswordStateConfig)
-    .config(PasswordResetInitStateConfig)
-    .config(PasswordResetFinishStateConfig)
-    .config(SettingsStateConfig)
-    <%_ if (authenticationType === 'session') { _%>
-    .config(SessionsStateConfig)
-    <%_ } _%>
```

Comment:
Don't have to register the ng2 states in a ng1 `.config()` block (although you _can_).
Instead of config blocks, the states are collected in an array and added to the `UIRouterModule.forChild()` NgModule, then imported into the root NgModule.